docs/_static/network/v1.yaml

```diff
@@ -70,6 +74,8 @@ paths:
               example:
         "404":
           description: "Transaction (or payload) wasn't found"
+        default:
+          $ref: '../common/error_response.yaml'
```

Comment:
default is for all 400+ status codes. In other apis the description for error returns is documented per path under path.get.description